### PR TITLE
Enforce templates attempt 2

### PR DESCRIPTION
**Pull request by MikeDev101**
Stupid workaround for Github desktop not signing commits.
Add issue templates to enforce new issue policy.

### CODE REVIEW

**Comment by showierdata9978** [comment]
good, your not using  conf.yaml to force it. 

SOME ISSUES, OR FEATURES HAVE TO HAVE A DIFFERENT LAYOUT.

**Comment by MikeDev101** [comment]
> SOME ISSUES, OR FEATURES HAVE TO HAVE A DIFFERENT LAYOUT.

As in...?

**Comment by showierdata9978** [comment]
>I didn't say that the template had to be followed like a fine-tooth comb, I'm just putting this here since templates help organize things.
I said that here
>good, your not using conf.yaml to force it.

> As in...?
Iv had to make issues that are not related to code, but something else that just needs a single line (like spotted typos, and issue is for a PR im making, where the issue only needs to be a single line, or maybe even just the title)

**Comment by CST1229** [comment]
This could use GH [issue forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms) (probably alongside a "blank issue" template, if that's needed).

**Comment by showierdata9978** [comment]
yeah iv seen projects stupidly enforce it, in a way where i cant report somth. (by making it so you cant bypass it)

**Comment by MikeDev101** [comment]
> This could use GH [issue forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms) (probably alongside a "blank issue" template, if that's needed).

> yeah iv seen projects stupidly enforce it, in a way where i cant report somth. (by making it so you cant bypass it)

There is always a blank option if the issue doesn't meet any template criteria (Below the template selection menu). But templates should be encouraged nonetheless.

**Comment by showierdata9978** [comment]
No, there is a conf.yml you can use that disables the blank issue

**Comment by MikeDev101** [comment]
> No, there is a conf.yml you can use that disables the blank issue

I'm not going to strictly require templates, which is why the config.yml is not present. I'm just saying that we should be using them, but have blanks as an option if needed. Blank/non-standard templates shouldn't be the first option provided.

**Comment by showierdata9978** [comment]
Yeah, that's why I said, Good no conf.yml

**Comment by MikeDev101** [comment]
> Yeah, that's why I said, Good no conf.yml

Rambling aside, is this good to merge? Still awaiting review from others.

**Comment by CST1229** [comment]
> Rambling aside, is this good to merge?

Still awaiting a response on

> This could use GH [issue forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms)

(as it seems like you're wanting the user to fill in blanks, which is what forms are for I think; for example, [the Scratch Addons repo uses them](https://github.com/ScratchAddons/ScratchAddons/issues/new/choose)). Otherwise, if you _do_ want to go with a text template, that's fine.

**Comment by showierdata9978** [comment]
> > Yeah, that's why I said, Good no conf.yml
> 
> Rambling aside, is this good to merge? Still awaiting review from others.

Yeah. I just wanted to make sure none of the mmc repos would change into the unusable issues of forced issue templates

**Review by showierdata9978 — APPROVED** [review]
Looks like the issue templates are good!

**Review by williamhorning — APPROVED** [review]
LGTM

**Review by CST1229 — APPROVED** [review]

**Review by JoshAtticus — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+## Platform info
+<!-- Please provide version info of the software you are using. -->
+**Meower Client**
+- Meower Client Version + Build Date: 
+
+<!-- Please fill out this section below.-->
+**Desktop/Mobile Device**
+ - OS: [Examples: Windows, Linux, macOS...]
+ - Version: [Examples: 22H2, 22.04, 13.7...]
```
[the remark this answers is not in the document]
Comment (mdwalters) [reply .github/ISSUE_TEMPLATE/bug_report.md:60]: linux 22.04 doesn't exist either

[the remark this answers is not in the document]
Comment (tnix100) [reply .github/ISSUE_TEMPLATE/bug_report.md:60]: > linux 22.04 doesn't exist either

No, but things like "Ubuntu 22.04" exist, and we're not going to have every Linux distro in the OS examples

[the remark this answers is not in the document]
Comment (JoshAtticus) [reply .github/ISSUE_TEMPLATE/bug_report.md:60]: Unlike macOS 13.7

[the remark this answers is not in the document]
Comment (JoshAtticus) [reply .github/ISSUE_TEMPLATE/bug_report.md:60]: Screenshot 2023-02-12 at 5 33 26 pm

**Review by silvxrcat — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (williamhorning) [reply .github/ISSUE_TEMPLATE/bug_report.md:60]: This is just an example, no need to scrutinize it this much.

**Comment by mdwalters** [comment]
yes we do

**Review by tnix100 — APPROVED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mdwalters) [reply .github/ISSUE_TEMPLATE/bug_report.md:60]: yes we do

**Pull request merged**